[2022-08-02 13:04:38,177] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: Demo.records_saving manual__2022-08-02T13:04:23.156641+00:00 [queued]>
[2022-08-02 13:04:38,189] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: Demo.records_saving manual__2022-08-02T13:04:23.156641+00:00 [queued]>
[2022-08-02 13:04:38,189] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-02 13:04:38,190] {taskinstance.py:1377} INFO - Starting attempt 1 of 1
[2022-08-02 13:04:38,190] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-02 13:04:38,208] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): records_saving> on 2022-08-02 13:04:23.156641+00:00
[2022-08-02 13:04:38,213] {standard_task_runner.py:52} INFO - Started process 10649 to run task
[2022-08-02 13:04:38,216] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'Demo', 'records_saving', 'manual__2022-08-02T13:04:23.156641+00:00', '--job-id', '301', '--raw', '--subdir', 'DAGS_FOLDER/demo.py', '--cfg-path', '/tmp/tmpzlx_i6gy', '--error-file', '/tmp/tmp299iz6v1']
[2022-08-02 13:04:38,217] {standard_task_runner.py:80} INFO - Job 301: Subtask records_saving
[2022-08-02 13:04:38,294] {task_command.py:371} INFO - Running <TaskInstance: Demo.records_saving manual__2022-08-02T13:04:23.156641+00:00 [running]> on host 31353b71d6cb
[2022-08-02 13:04:38,383] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=Demo
AIRFLOW_CTX_TASK_ID=records_saving
AIRFLOW_CTX_EXECUTION_DATE=2022-08-02T13:04:23.156641+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-08-02T13:04:23.156641+00:00
[2022-08-02 13:04:38,385] {python.py:173} INFO - Done. Returned value was: 0
[2022-08-02 13:04:38,415] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=Demo, task_id=records_saving, execution_date=20220802T130423, start_date=20220802T130438, end_date=20220802T130438
[2022-08-02 13:04:38,469] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-02 13:04:38,513] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
